src/Client/PullPayment.php

```diff
@@ -0,0 +1,336 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Client;
+
+class PullPayment extends AbstractClient
+{
+    /*
+    |--------------------------------------------------------------------------
+    | Management Pull Payment Methods.
+    |--------------------------------------------------------------------------
+    |
+    | For management to get, create and archive a pull payment.
+    | Approve, cancel or mark a payout as paid.
+    |
+    */
+
+    /**
+    * Get Store's Pull Payments
+    *
+    * @return string $id
```

Comment:
1. should be @param and not @result 
2. there are too many params listed, the function only accepts two parameters
3. we do not list the params here as they are already typehinted in the function signature, only if there is additional information or logic needed we list them explicitly

Reply:
I was thinking it would be helpful to list what fields the API returns in JSON? These aren't the same as the params. Let me know I can remove them, maybe this is more helpful on the "usage" examples then right in the class?